cypress/support/network_commands.js

```diff
@@ -6,17 +6,21 @@
  */
 
 Cypress.Commands.add('importJSONMapping', (filepath, hostname = 'localhost', port = '9200') => {
+  const parseIndex = (indexValue) => {
+    const index = indexValue.index
+    const settings = indexValue.settings
+    const mappings = indexValue.mappings
+    const aliases = indexValue.aliases
+    return { index, body: { settings, mappings, aliases } }
+  }
+
   cy.readFile(filepath, 'utf8').then((str) => {
     const strSplit = str.split('\n\n')
-    cy.wrap(strSplit).each((element) => {
+    strSplit.forEach((element) => {
       const json = JSON.parse(element)
       if (json.type === 'index') {
-        const index = json.value.index
-        const settings = json.value.settings
-        const mappings = json.value.mappings
-        const aliases = json.value.aliases
-        const body = { settings, mappings, aliases }
-        cy.request({ method: 'PUT', url: `${hostname}:${port}/${index}`, body: body, failOnStatusCode: false }).then((response) => {
+        const indexContent = parseIndex(json.value)
+        cy.request({ method: 'PUT', url: `${hostname}:${port}/${indexContent.index}`, body: indexContent.body, failOnStatusCode: false }).then((response) => {
```

Comment:
this line is too long, could we setup prettier for project to format the code automatically. 

refer to 
https://github.com/elastic/kibana/blob/master/.prettierrc

no need to address in this PR, just create an open issue to track

Reply:
Sounds good, I'll open an issue as you suggested so that it can get added in a future PR.